src/main/java/com/raisetech/mybatis/service/RamenServiceImpl.java

```diff
@@ -31,11 +31,17 @@ public List<Ramen> getRamenList(Integer point) {
     }
   }
 
-  public void update(int id, String name, String address, Integer avePrice, Integer point) {
+  public void create(Ramen ramen) {
+    Ramen ramenEntity = Ramen.newInstance(ramen.getName(), ramen.getAddress(), ramen.getAvePrice(), ramen.getPoint());
+    ramenMapper.create(ramenEntity);
+    ramen.setId(ramenEntity.getId());
+  }
```

Comment:
これでもよいですが、私なら以下のように登録した内容を戻り値として返します。
```suggestion
  public Ramen create(Ramen ramen) {
    Ramen ramenEntity = Ramen.newInstance(ramen.getName(), ramen.getAddress(), ramen.getAvePrice(), ramen.getPoint());
    ramenMapper.create(ramenEntity);
    return ramenEntity;
  }
```

Reply:
ただ、今の実装のままでもよいです 👍